gateway-test/pom.xml

```diff
@@ -172,6 +172,12 @@
             <groupId>io.rest-assured</groupId>
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
Why do we need this? This looks like a test only exclusion?

Reply:
javax.mail has a different version of javax.activation causing builds to breaks.

Reply:
Exclude it at the top level pom then? Where rest-assured is defined?

Reply:
That can be done but do we want to exclude this dependency for all the projects ? it should be OK here given that this is for test, it should not matter since we do check for unit tests before builds. Let me try that.